gadget/install/install.go

```diff
@@ -551,6 +550,13 @@ func MountVolumes(onVolumes map[string]*gadget.Volume, encSetupData *EncryptionS
 				continue
 			}
 			mntPt := filepath.Join(boot.InitramfsRunMntDir, part.Name)
```

Comment:
this is not related to this but this code is using part.Name while the code in Run was using part.Label, this is confusing I think and will bite us. What happens if we use label here? except making a special case for ubuntu-seed-null

Reply:
What I see is that functions that access the mounted folders use `InstallHostWritableDir()` and `InitramfsUbuntu*Dir`, which hard-code paths that include "ubuntu-seed", "ubuntu-data", etc. So I think that actually the way forward should be to use those paths instead of using names or labels. I've changed it in that direction now.